packages/mobile/src/transactions/transferFeedUtils.ts

```diff
@@ -52,45 +56,64 @@ export function getTransferFeedParams(
     case TokenTransactionType.Faucet: {
       title = t('feedItemFaucetTitle')
       info = t('feedItemFaucetInfo', {
-        context: !DEFAULT_TESTNET ? 'missingTestnet' : null,
+        context: !DEFAULT_TESTNET ? 'noTestnet' : null,
         faucet: DEFAULT_TESTNET ? _.startCase(DEFAULT_TESTNET) : null,
       })
       break
     }
     case TokenTransactionType.InviteSent: {
-      const inviteeE164Number = addressToE164Number[address]
-      const inviteeRecipient = inviteeE164Number ? recipientCache[inviteeE164Number] : undefined
       title = t('feedItemInviteSentTitle')
       info = t('feedItemInviteSentInfo', {
-        context: !inviteeE164Number ? 'missingInviteeDetails' : null,
-        nameOrNumber: inviteeRecipient ? inviteeRecipient.displayName : inviteeE164Number,
+        context: !nameOrNumber ? 'noInviteeDetails' : null,
+        nameOrNumber,
       })
       break
     }
-
     case TokenTransactionType.InviteReceived: {
       title = t('feedItemInviteReceivedTitle')
       info = t('feedItemInviteReceivedInfo')
       break
     }
+    case TokenTransactionType.Sent: {
+      title = t('feedItemSentTitle', {
+        context: !nameOrNumber ? 'noReceiverDetails' : null,
```

Comment:
Nit: could use an enum for the contexts to prevent typos?

Reply:
yes, managing translation strings is a bit too manual and error prone, I've mentioned [js-lingui](https://github.com/lingui/js-lingui) in the past and want to propose using it.
It would solve this kind of issue as it automatically extracts strings from the source code.
Though i18next has something similar which could be worth looking at.